controls/SV-221584.rb

```diff
@@ -21,4 +21,14 @@
   tag legacy: ["SV-57639","V-44805"]
   tag cci: ["CCI-002605"]
   tag nist: ["SI-2 c"]
+
+  current_chrome_version_command  = <<-EOH
+    # $(Get-Package -Name "Google Chrome").Version
+    $version = (Get-Item (Get-ItemProperty 'HKLM:\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\chrome.exe').'(Default)').VersionInfo.ProductVersionwerwe
+    echo $version
+  EOH
+
+  describe powershell(current_chrome_version_command) do
+    its('stdout') { should cmp >= input('google_chrome_version')  }
```

Comment:
Looking at the automated check, even though the Google Chrome on the GitHub runner is not hardened, it should be 105.0.5195.102 based on https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md. Is there a reason it is not passing in the pipeline test then? This is the one I would expect to pass and it did pass in some previous runs.

Reply:
current_chrome_version_command == '105.0.5195.102' # stdout string
google_chrome_version == '74.0.0' # string
current_chrome_version_command >= google_chrome_version is False since they are strings.

Reply:
Is there any way **current_chrome_version_command** can output as just a string then?

Reply:
Before changing much of what you have, the first thing I would try is the simplest, see what results you get when you change 
`    its('stdout') { should cmp >= input('google_chrome_version')  }`
to 
`    its('stdout') { should be >= input('google_chrome_version')  }`